azure-pipelines-20230430.yml

```diff
@@ -220,39 +219,3 @@ stages:
           - script: |
               grep "testcase" */target/surefire-reports/*.xml */*/target/surefire-reports/*.xml | awk -F'"' ' { print $6,$4,$2 } ' | sort -nr | head -n 100
             displayName: Top 100 long-running testcases
-      - job: IT
```

Comment:
IT will still be running with spark2.4 as per the docker demo setup, and moved to GH actions